BuildCliComponents.cmd

```diff
@@ -40,13 +43,13 @@ if NOT exist "%DotNet%" (
 echo Building CLI-based components
 pushd %~dp0src\cli\Microsoft.DotNet.xunit.performance.runner.cli
 call %DotNet% restore
-call %DotNet% build -c Release
-call %DotNet% pack -c Release -o %OutputDirectory%
+call %DotNet% build -c %BuildConfiguration%
+call %DotNet% pack -c %BuildConfiguration% -o %OutputDirectory%
 popd
 pushd %~dp0src\cli\Microsoft.DotNet.xunit.performance.analysis.cli
 call %DotNet% restore
-call %DotNet% build -c Release
-call %DotNet% pack -c Release -o %OutputDirectory%
+call %DotNet% build -c %BuildConfiguration%
+call %DotNet% pack -c %BuildConfiguration% -o %OutputDirectory%
```

Comment:
Gotcha.  I do want us to be able to build debug nuget packages for when we need them for testing.  However, you reminded me of another issue that we do need to solve, so I've filed it at #141.